templates/firefox-ios.xliff

```diff
@@ -4123,6 +4231,62 @@ Private Tab</source>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.NightMode.Subtitle.v132" xml:space="preserve">
+        <source>Night Mode</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Night Mode tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.Print.Subtitle.v132" xml:space="preserve">
+        <source>Print</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Report Print tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.Print.Title.v132" xml:space="preserve">
+        <source>Print</source>
+        <note>On the main menu, the accessibility label for the action that will take the user to the Print module in the application.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReaderView.Off.Title.v132" xml:space="preserve">
+        <source>Turn off Reader View</source>
+        <note>On the main menu, the accessibility label for the action that will turn the reader view on for the current website.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReaderView.On.Title.v132" xml:space="preserve">
+        <source>Turn on Reader View</source>
+        <note>On the main menu, the accessibility label for the action that will turn the reader view on for the current website.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReaderView.Subtitle.v132" xml:space="preserve">
+        <source>Reader View</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Reader View tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReportBrokenSite.Subtitle.v132" xml:space="preserve">
+        <source>Report</source>
+        <note>On the main menu, a string below the Tool submenu accessibility label, indicating what kind of tools are available in that menu. This string is for the Report Broken Site tool.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.ReportBrokenSite.Title.v132" xml:space="preserve">
+        <source>Report Broken Site</source>
+        <note>On the main menu, the accessibility label for the action that will take the user to the site where they can report a broken website to our web compatibility team.</note>
+      </trans-unit>
+      <trans-unit id="MainMenu.Submenus.Tools.AccessibilityLabels.Share.Subtitle.v132" xml:space="preserve">
+        <source>Share</source>
```

Comment:
What does "Share" refer to? Is it a noun or a verb? Localizers need more context in localization comments to translate this correctly